programming_guide/section-2/section-2c/README.md

```diff
@@ -95,18 +96,25 @@ of0 = object_fifo
         A,
         B,
         3,
-        T.memref(256, T.i32()),
-        [
-            (m, k),
-            (mtk // k, m * k),
-            (k, 1),
-        ],
+        T.memref((4, 8), T.i8()),
         [
-            [
-                (m, k),
-                (mtk // k, m * k),
-                (k, 1),
-            ]
+            (2, 16),
+            (3, 2),
         ],
     )
 ```
+The access pattern of the transformation can be written as:
+```
+for(int i = 0; i < 2; i++)      # size_1
+    for(int j = 0; j < 3; j++)  # size_0
+        # access/store element at/to index:
+        (
+            i * 16  # stride_1 
+            + j * 2 # stride_0
+        )
+```
+and further represented as in the image below,
+
+<img height="300" src="./../../assets/DataLayoutTransformation.svg">
+
+Other examples containing data layout transformations are available in the [programming_examples](../../../programming_examples/). A few notable ones are [matrix_vector_multiplication](../../../programming_examples/basic/matrix_multiplication/matrix_vector/) and [matrix_multiplication_single_column](../../../programming_examples/basic/matrix_multiplication/single_column/).
```

Comment:
We deprecated the single_column design. Maybe highlight single_core or whole_array.